src/frontend/src/pages/FinancePage/ReimbursementRequestDetailPage/SubmitToSaboModal.tsx

```diff
@@ -53,6 +54,21 @@ const SubmitToSaboModal = ({ open, setOpen, reimbursementRequest }: SubmitToSabo
     setOpen(false);
   };
 
+  const CopyToClipboardButton = ({ msg }: { msg: string }) => {
```

Comment:
I would make this a component in its own file and put it in the components folder